### PR TITLE
chore: add catalog-info.yml

### PR DESCRIPTION
**Pull request by ichintanjoshi**
## Description

- added catalog-info.yaml

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ichintanjoshi! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  description: "A Piral shell application for use with Open edX MFEs."
+  links:
+    - url: "https://github.com/ichintanjoshi/frontend-app-shell"
+      title: "frontend-app-shell"
+      icon: "GitHub"
+  annotations:
+    openedx.org/arch-interest-groups: ""
+    openedx.org/release: "main"
+spec:
+  owner: group:openedx-unmaintained
```
Comment (feanil) [inline catalog-info.yaml]: @arbrandes This seems like it's new enough that we should have a maintainer.  Are you the right person to be the maintainer for this repo?

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+metadata:
+  name: 'frontend-app-shell'
+  description: "A Piral shell application for use with Open edX MFEs."
+  links:
+    - url: "https://github.com/ichintanjoshi/frontend-app-shell"
+      title: "frontend-app-shell"
+      icon: "GitHub"
+  annotations:
+    openedx.org/arch-interest-groups: ""
+    openedx.org/release: "main"
```
Comment (feanil) [inline catalog-info.yaml]: You should not put in this value by default, it should only go in if the repo is going to be tagged for releases,  it should match the value in the `openedx.yaml` file in the repo if it exists. If it doesn't exist, then this annotation should not be added.

File: catalog-info.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'frontend-app-shell'
+  description: "A Piral shell application for use with Open edX MFEs."
+  links:
+    - url: "https://github.com/ichintanjoshi/frontend-app-shell"
+      title: "frontend-app-shell"
+      icon: "GitHub"
```
Comment (feanil) [inline catalog-info.yaml]: This is the wrong link and we don't need to link to the github repo that happens automatically.

Comment (ichintanjoshi) [reply catalog-info.yaml]: @feanil  have updated

File: catalog-info.yaml (hunk 2 above)
Comment (ichintanjoshi) [reply catalog-info.yaml]: @feanil  have removed

File: catalog-info.yaml (hunk 1 above)
Comment (arbrandes) [reply catalog-info.yaml]: Yeah, sign me up!

File: catalog-info.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'frontend-app-shell'
+  description: "A Piral shell application for use with Open edX MFEs."
+spec:
+  owner: group:openedx-unmaintained
```
Comment (feanil) [inline catalog-info.yaml]:
```suggestion
  owner: user:arbrandes
```

File: catalog-info.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'frontend-app-shell'
+  description: "A Piral shell application for use with Open edX MFEs."
+spec:
+  owner: user:arbrandes
+  type: 'service'
+  lifecycle: 'production'
```
Comment (feanil) [inline catalog-info.yaml]:
```suggestion
  lifecycle: 'experimental'
```

File: catalog-info.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: 'frontend-app-shell'
+  description: "A Piral shell application for use with Open edX MFEs."
```
Comment (feanil) [inline catalog-info.yaml]:
```suggestion
  description: "A shell application for use with Open edX MFEs."
```

**Review by feanil — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@ichintanjoshi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-shell&entry.867055334=https://github.com/openedx/frontend-app-shell/pull/9&entry.1484655318=https://github.com/ichintanjoshi&entry.752974735=2024-05-29+19:33&entry.1917517419=2024-06-20+17:43&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**